beets/util/functemplate.py

```diff
@@ -318,9 +318,11 @@ def __init__(self, string):
 
     # Common parsing resources.
     special_chars = (SYMBOL_DELIM, FUNC_DELIM, GROUP_OPEN, GROUP_CLOSE,
-                     ARG_SEP, ESCAPE_CHAR)
+                     ESCAPE_CHAR)
     special_char_re = re.compile(r'[%s]|$' %
                                  u''.join(re.escape(c) for c in special_chars))
+    escapable_chars = (SYMBOL_DELIM, FUNC_DELIM, GROUP_CLOSE, ARG_SEP)
```

Comment:
I left `ARG_SEP` deliberately in the list of escapable chars in order to allow both escaped and unescaped commands outside a function (`-f 'foo, $bar'` would work exactly the same as `-f 'some$, $thing'`).

I'm undecided about it, though: on one hand, it keeps the syntax backwards compatible-ish, but on the other hand introduces some ambiguity.